tests/common/jcli_wrapper/mod.rs

```diff
@@ -15,6 +15,12 @@ use super::process_utils::output_extensions::ProcessOutput;
 use std::collections::BTreeMap;
 use std::path::PathBuf;
 
+#[derive(PartialEq)]
+pub enum Discrimination {
```

Comment:
I'd call it `Profile` or maybe `BlockchainSelection` for clarity, but it's a little matter.

Reply:
Unless "address discrimination" is an established term that I've missed.